### ISSUE TITLE
Arabic strings are not "naughty" for arabic websites

### ISSUE THREAD

**Issue by MohammedRashad** [minimaxir/big-list-of-naughty-strings#216]
I saw that the list has Arabic text, why is this considered "naughty" ?
for example all social media and most programming languages support utf-8 strings, which include Arabic.

**Comment by FlyingBlueMonkey** [comment]
I believe this is covered in the readme:   "_The Big List of Naughty Strings is an evolving list of strings which have a high probability of causing issues when used as user-input data. This is intended for use in helping both automated and manual QA testing_; "

in other words, these are strings to ensure that applications handle the strings / layouts correctly when using them.  As is commented in the file itself:  "
#	Right-To-Left Strings
#
#	Strings which contain text that should be rendered RTL if possible (e.g. Arabic, Hebrew)
"

**Comment by ssokolow** [comment]
See, for example, Lord i/o's [Text Editing Hates You Too](https://lord.io/blog/2019/text-editing-hates-you-too/). It touches on mixing LTR and RTL text.

**Comment by MohammedRashad** [comment]
@FlyingBlueMonkey 
Makes sense now, but I think this we should break the current list to more specific lists, what do you think?

@ssokolow 
I'm checking the lord.io blog post. 
Thanks for the addition.
